WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -66,15 +67,39 @@
             <% else %>
               <%= alert :success, t('registrations.accepted') %>
             <% end %>
+            <div>
+              <% if @competition.connected_stripe_account_id %>
+                <% paid_entry_fees = humanized_money_with_symbol @registration.paid_entry_fees %>
+                <% outstanding_entry_fees = humanized_money_with_symbol @registration.outstanding_entry_fees %>
+                <% entry_fee = humanized_money_with_symbol @registration.entry_fee %>
+                <% if @registration.outstanding_entry_fees > 0 %>
+                  <% if @registration.paid_entry_fees > 0 %>
+                    <%= alert :info, t('registrations.entry_fees_partially_paid', paid: paid_entry_fees, total_entry_fee: entry_fee) %>
+                  <% end %>
+                  <%= form_tag competition_process_payment_path do %>
+                    <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+                      data-key="<%= ENVied.STRIPE_PUBLISHABLE_KEY %>"
+                      data-name="World Cube Association"
+                      data-description="Registration fee"
+                      data-amount="<%= @registration.outstanding_entry_fees.cents %>"
+                      data-currency="<%= @registration.entry_fee_currency_code %>"
```

Comment:
Just curious... would we be able to get the currency code from `@registration.outstanding_entry_fees` instead? I feel like always interacting with the Money object would minimize our "api surface" in a nice way.